front/tests/signup_test.html

```
@@ -0,0 +1,10 @@
+<div id="test-container">
+    <form id="signup_form">
+        <input type="text" id="username">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)